CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
 add_subdirectory(3rdparty/glfw/)
 target_include_directories(chani PRIVATE 3rdparty/glfw/include/)
 target_sources(chani PRIVATE 3rdparty/imgui/examples/libs/gl3w/GL/gl3w.c)
-target_link_libraries(chani glfw)
+set(OpenGL_GL_PREFERENCE GLVND)
```

Comment:
Can you put these in a conditional block for Linux (and/or WSL)? Like we have for Mac OS X further down. They break the CMake build on Mac OS X.

Reply:
Will do !

Reply:
Update done.